Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "userfaultfd"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
Minor version bump because the would-be breaking change is behind a new cargo feature